cpp/src/io/parquet/page_data.cu

```diff
@@ -805,7 +805,7 @@ inline __device__ void gpuOutputInt96Timestamp(volatile page_state_s *s, int src
         // Convert from Julian day at noon to UTC seconds
         day = (day - 2440588) * (24 * 60 * 60); // TBD: Should be noon instead of midnight, but this matches pyarrow
         if (s->col.ts_clock_rate)
-            ts = (day * s->col.ts_clock_rate) + (nanos + (499999999 / s->col.ts_clock_rate)) / (1000000000 / s->col.ts_clock_rate); // Output to desired clock rate
+            ts = (day * s->col.ts_clock_rate) + nanos / (1000000000 / s->col.ts_clock_rate); // Output to desired clock rate
```

Comment:
Variable name _day_ here is misleading as after the conversion in line 806, this variable holds the number of seconds since Jan. 1st, 1970.

Reply:
Updated the name.